notebooks/article_1.py

```diff
@@ -0,0 +1,1142 @@
+f = d[["has_VC", "has_angel", "has_roundA", "has_roundB", "has_roundC", "has_roundD"]]
+
+# Melt the DataFrame to long format for use with countplot
+melted_f = pd.melt(f)
+
+# Map the binary values to strings
+melted_f['value'] = melted_f['value'].map({1: 'acquired', 0: 'closed'})
+
+# Create the countplot with horizontal bars
+fig, ax = plt.subplots(figsize=(10, 6))  # Adjust the figure size as needed
+sns.countplot(data=melted_f, y='variable', hue='value', orient='h', palette="rainbow")
+
+# Update the y-axis label with adjusted position
+ax.set_ylabel("Investment Types", labelpad=20)  # Adjust labelpad for position
+ax.set_xlabel("Startup Count", fontsize=10, labelpad=25)
+
+# Create the title manually and adjust its position
+ax.text(0.2, 1.05, "Investment Types for Acquired Startups", fontsize=15, transform=ax.transAxes, fontweight='bold')  # Adjust the x and y values
+
+# Update the legend
+handles, labels = ax.get_legend_handles_labels()
+ax.legend(handles, ['Closed', 'Acquired'], title='Status')
+
+plt.show()
+
+
+# It shows that the majority of startups that received round D funding were acquired rather than closed. 
+# 
+# Investments from venture capital (VC) and rounds A, B, and C are more commonly associated with startups that were acquired, suggesting these types of investments might be linked to a higher likelihood of success.
+# 
+# Now that we have confirm the importance of type of invesment is a factor for a startup to succed or not, let's analyse another variable that was highly correlated with our target.
+
+#  #### What's the average funding for acquired vs. closed startups?
+
+# In[103]:
+
+
+# Define a list of pastel color codes
+colors = ['#1f77b4', '#d62728']
+
+# Filter out extreme outliers from the data
+max_value = data_df['funding_total_usd'].quantile(0.99) 
+filtered_data = data_df[data_df['funding_total_usd'] < max_value]
+
+# Create a histogram with the filtered data
+fig = px.histogram(data_frame=filtered_data, x='funding_total_usd', color='status',
+                   labels={'funding_total_usd': 'Total Funding (USD in Millions)'},
+                   color_discrete_sequence=colors)  
+                   
+# Update layout for better visualization
+fig.update_layout(
+    title={
+        'text': 'Distribution of Total Funding : Acquired vs Closed Startups',
+        'y': 0.95,  # You can adjust this for vertical position
+        'x': 0.5,  # You can adjust this for horizontal position
+        'xanchor': 'center',
+        'yanchor': 'top',
+        'font': {
+            'size': 17,
+            'color': 'black',  # You can change the color if you want
+            'family': 'Arial, sans-serif', # You can change the font family if you want
+        },
+    },
+    xaxis_title='Total Funding (USD in Millions)',
+    yaxis_title='Frequency',
+)
+# Format the x-axis tick labels to display values in millions
+fig.update_xaxes(tickformat=".2s", exponentformat="none")
+
+# Show the figure
+fig.show()
```

Comment:
In this figure, the legend has `0` and `1`. Instead write `Acquired` and `closed`.